client/landing/stepper/index.tsx

```diff
@@ -143,6 +143,11 @@ window.AppBoot = async () => {
 						{ config.isEnabled( 'gdpr-banner' ) && (
 							<AsyncLoad require="calypso/blocks/gdpr-banner" placeholder={ null } />
 						) }
+						<AsyncLoad
+							require="calypso/components/global-notices"
+							placeholder={ null }
+							id="notices"
+						/>
```

Comment:
This seems fitting here, but cc'ing @alshakero and @Automattic/vertex team as well since were adding this to a much more generic stepper page.

Reply:
I was curious about this. Very interesting implementation that's neat to see. Two related questions/thoughts. I should note I don't have strong feelings about how to approach this.

1) Is there an expectation that the notices component will be loaded on other pages? If not (or not yet), could this component be loaded in a more conventional way on the launchpad screen? 

2) If we are loading the async notices component here, would it make any sense to put the logic for if/when to load it here too? For example, could you move the logic to detect emailVerified and dispatch the email notice here vs launchpad/index.tsx? If we did that, and if in the future other stepper pages need the email notice, that would already be handled. If other stepper pages need other notices, you could similarly detect those params and conditionally adjust the notice message here.

Reply:
> Is there an expectation that the notices component will be loaded on other pages? If not (or not yet), could this component be loaded in a more conventional way on the launchpad screen?

I think there is a good case for it to live at this level vs. nested/isolated down at the launchpad. It doesn't seem like there is any current need for notices here, but there may be in the future. But really I think this sort of component makes sense at this higher level in component the hierarchy, enabling this functionality more broadly as opposed to just on specific pages when we need them.

Reply:
> Is there an expectation that the notices component will be loaded on other pages? If not (or not yet), could this component be loaded in a more conventional way on the launchpad screen?

As @Addison-Stavlo & @alshakero mentioned, I wanted to add the `<GlobalNotices />` in a location outside of any flows/steps since notifications would be handled separately from the actual stepper framework and this component is supposed to be considered "global" on any page.

> If we are loading the async notices component here, would it make any sense to put the logic for if/when to load it here too?

Yes, that would work as well. I thought about that, but I was unsure if it was going to be used and figured I could move it outside of the launchpad when necessary. Either way, it can easily be moved (now or later) to `/stepper/index.tsx`. Does anyone have any strong opinions about moving the logic to `/stepper/index.tsx` or keeping it inside `launchpad/index.tsx`?

Reply:
No strong feelings on my side. And as you noted, easy enough to move later if warranted.